dm/master/task_basic.yaml

```diff
@@ -0,0 +1,53 @@
+---
+name: test # global unique
+task-mode: all  # full/incremental/all
+
+target-database:
+  host: "192.168.0.1"
+  port: 4000
+  user: "root"
+  password: ""
+
+mysql-instances:             # one or more source database, config more source database for sharding merge
+  -
+    source-id: "instance118-4306" # unique in all instances, used as id when save checkpoints, configs, etc.
+    black-white-list:  "instance"
+
+    # `mydumper-thread` is alias for `threads` in `mydumper` config, and it's priority is higher than `threads`
```

Comment:
in _task_basic_ file, give it a meaningful comment?

Reply:
update https://github.com/pingcap/dm/pull/309/commits/1cde56519be4c5da87635a3406aeb6c982a90d21